src/coreclr/jit/loopcloning.h

```diff
@@ -421,10 +435,12 @@ struct LC_Ident
         Var,
         ArrLen,
         Null,
+        ClassHandle,
+        Indir,
     };
 
     LC_Array  arrLen;   // The LC_Array if the type is "ArrLen"
-    unsigned  constant; // The constant value if this node is of type "Const", or the lcl num if "Var"
+    ssize_t   constant; // The constant value if this node is of type "Const", or the lcl num if "Var"
```

Comment:
Maybe this should be a `union`? Seems like that would eliminate a lot of casts (and make this struct smaller).

Reply:
Yeah, I thought about doing that... makes sense.